expo/tests/scheduler/test_schedule_acts.py

```diff
@@ -1,7 +1,7 @@
 from django.core.urlresolvers import reverse
 from django.test import (
     Client,
-    TestCase,
+    TestCase
```

Comment:
Doh. I was looking for a trailing comma on the final item in the other import. This is a good habit to get into, since it means that the next person adding an import isn't going to get caught on a missing comma. 

Not going to get hung up on this one, though.